hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconUtils.java

```diff
@@ -245,6 +246,16 @@ public InputStream makeHttpCall(CloseableHttpClient httpClient, String url)
     }
   }
 
+  public InputStream makeHttpCall(URLConnectionFactory connectionFactory,
+                                  String url)
+      throws IOException {
+
+    java.net.URLConnection urlConnection =
```

Comment:
NIT: We can import java.net.URLConnection and java.net.URL

Reply:
Done